CLI/src/Writer.cs

```diff
@@ -141,7 +141,14 @@ public class XMLWriter : AbstractErrorWriter {
             writer.WriteStartElement("MESSAGE");
             writer.WriteElementString("MSGNUMBER",
                 "TC-" + error.Info.Code.ToString().PadLeft(5, '0') + AsIBMSuffix((int) error.Info.Severity));
-            writer.WriteElementString("MSGLINE", error.Line.ToString());
+            if (error.LineStart == error.LineEnd)
+            {
+                writer.WriteElementString("MSGLINE", error.LineStart.ToString());
+            }
+            else
+            {
+                writer.WriteElementString("MSGLINE", $"{error.LineStart}:{error.LineEnd}");
```

Comment:
Is that valid? How to know if this is correct?

Reply:
I don't know. We have to find how this format is used by our clients.

Reply:
There is [this page](https://www.ibm.com/docs/en/zvm/7.2?topic=areas-msgline) in the IBM's documentation saying that [RSCS](https://www.ibm.com/docs/en/zvm/7.1?topic=features-rscs-networking-zvm) creates chain of MSGLINE for multiple lines.

Reply:
Hum interesting find, but I don't think it's _that_ MSGLINE. The XML file written here is used to transfer diagnostics to our private RDZ editor (instead of using LSP).

Reply:
Apparently [this is the schema](https://www.ibm.com/docs/en/adfz/developer-for-zos/15.0.0?topic=mechanism-error-feedback-xml-schema) we need to follow.